frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -52,40 +53,12 @@ function ReaderFooter:init()
         book_time_to_read = true,
         chapter_time_to_read = true,
     }
-    local text_default
-    if self.settings.all_at_once then
```

Comment:
Was this a bunch of unused code or did you forget to put it somewhere else/rewrite it?